File: app/src/main/java/com/udacity/gradle/builditbigger/Models/Country.java
package com.udacity.gradle.builditbigger.Models;

public class Country {
    private String emoji;
    private String twoDigit;

    public Country(String twoDigit, String emoji){
        this.emoji = emoji;
        this.twoDigit = twoDigit;
    }

    public String getEmoji() {
        return emoji;
    }

    public String getTwoDigit() {
        return twoDigit;
    }
}

File: app/src/main/java/com/udacity/gradle/builditbigger/models/Country.kt
package com.udacity.gradle.builditbigger.models

/**
 * A class used to represent the country the user is from
 *
 * @property twoDigit is the two digit code used to refer to a country
 * @property emoji is the string representation of the nation's flag
 */

data class Country(val twoDigit: String, val emoji: String)